bin/internal/shared.bat

```diff
@@ -157,7 +158,13 @@ GOTO :after_subroutine
     IF "%ERRORLEVEL%" EQU "2" (
       EXIT 1
     )
+    SET max_dart_sdk_retries = "3"
```

Comment:
Change this to `SET max_dart_sdk_retries=3`

(The variable will not actually be set if there is whitespace around the `=`)

Reply:
I love Batch scripting! 🫠

Reply:
🤦‍♂️